utilities/pipelines/resourceRemoval/Initialize-DeploymentRemoval.ps1

```diff
@@ -74,6 +74,10 @@ function Initialize-DeploymentRemoval {
             'Microsoft.OperationalInsights/workspaces/linkedServices',
             'Microsoft.OperationalInsights/workspaces',
             'Microsoft.KeyVault/vaults',
+            'Microsoft.Authorization/policyExemptions',
```

Comment:
Just for my knowledge, what was the behavior before? As I don't recall we had issues with removal?

Reply:
We did not because we did not have a test cases that deployed them all together. No we do - and I noticed that the pipeline would complain if you try to remove a policySet that is still assigned ;)